src/components/accessories/patientSummary/patientSummaryByDate/PatientSummaryByDate.tsx

```diff
@@ -37,7 +37,7 @@ const PatientSummaryByDate: FunctionComponent<TProps> = ({
         {!isLoading ? (
           <Table
             rowData={renderSummary(summaryData, dateFields, labels, medicals)}
-            compareRows={dateComparator}
+            dateFields={dateFields}
```

Comment:
I check the code and also the ui and it OK for me. At least you should remove unused dateComparator import `import { dateComparator } from "../../../../libraries/sortUtils/sortUtils";`  from the files that you changed.